client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -110,6 +258,18 @@
 			{#if selectedPiece}
 				<div></div>
 			{/if}
+			{#if orders}
```

Comment:
Idea for a separate component:

`<ArenaLegend>` which just renders the legend, no js code, which would just shuttle some of the extra lines from this file to another one.